app/views/votes/_toggle_like.html.erb

```diff
@@ -3,8 +3,15 @@
   liked = user_signed_in? && current_user.voted_for?(votable)
 %>
 
-<%= form_tag vote_path, method: (liked ? :delete : :post) do %>
+<%= form_tag vote_path, method: (liked ? :delete : :post), class: "like-form" do %>
```

Comment:
I was thinking about doing the opposite of adding HTML classes like these, actually. I don't think we should be naming them after their backend models, but their frontend functionality. That way we could reuse them across more views.

That said, I don't care much about this at the moment to force a change.